wallet/src/main/AndroidManifest.xml

```diff
@@ -20,6 +45,44 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <activity
+            android:name=".PresentationActivity"
+            android:exported="true"
+            android:label="@string/app_name_presentation"
+            android:theme="@style/Theme.IdentityCredential">
+        </activity>
+
+        <service
+            android:name=".NfcEngagementHandler"
+            android:exported="true"
+            android:label="@string/nfc_engagement_service_desc"
+            android:permission="android.permission.BIND_NFC_SERVICE">
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
```

Comment:
Do we need NDEF_DISCOVERED? I think that's only for the reader...

Reply:
good point, thanks